styles.css

```diff
@@ -145,3 +145,276 @@ h2 {
   color: #396df2;
   display: block;
 }
+
+.middle-section-container2 {
+  display: grid;
+  width: 350px;
+  height: 700px;
+  padding: 15px;
+  border-radius: 20px;
+  border: 2px solid white;
+  align-items: center;
+  margin-bottom: 100px;
+  background-color: white;
+}
```

Comment:
- [ ] Kindly fix your design to match the one in the Figma template.

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/186270799-b3925090-079a-493f-8d74-6f07c7bd9245.png) | ![image](https://user-images.githubusercontent.com/53402600/186270763-6941f632-dc62-42d6-8702-74cac573f320.png) |

- [ ] [OPTIONAL] It will be better if you remove the static values for each div and let it take the default width size to grow and shrink automatically without adjusting it and this will help you in the desktop version.